bundles/org.openhab.binding.salus/.gitignore

```diff
@@ -0,0 +1 @@
+*.http
```

Comment:
why is this needed? maybe do not inlude....

Reply:
I don't want to commit files that has extension `http`

Reply:
clear, but is this needed for everybody checking out the repo?

Reply:
I have a lot of http files (it's like curl) that I've used to check salus API. I don't want to commit them cause they contains my username/password/token, but still I want to have them with the project

Reply:
btw it will only apply to files in salus folder, not in all folders

Reply:
Why? This gitignore is only for this folder not whole project and it will only affect http files for salus binding. If this entry would be removed then I would be unable to store http files in this project

Reply:
You can have the file in your development environment, no problem,  just leave it out of this PR.

Reply:
It will not work because my file will always appear as "commitable" and because of it at some point of time I might commit it by mistake. 

Whats the reason that this file should be removed?

Reply:
As it is specific to your dev environment. We managed to keep such files out of 400+ bindings ;-) i thik we will detect that in future PR's

Reply:
> I have a lot of http files (it's like curl) that I've used to check salus API.

Why not just keep them in a directory outside the project?

Reply:
> Why not just keep them in a directory outside the project?

Because it's run by IntelliJ and it's easier if it's here

> As it is specific to your dev environment. We managed to keep such files out of 400+ bindings ;-) i thik we will detect that in future PR's

`*.iml` is also specific to dev env. 

I would understand why you don't want to have it in main `.gitignore`, but I do not understand why not have it in binding specific `.gitignore`

Reply:
This answered before, please remove.

Reply:
this makes no sense